app/src/main/java/se/nullable/flickboard/model/layouts/PTANDMessagEase.kt

```diff
@@ -0,0 +1,115 @@
+package se.nullable.flickboard.model.layouts
```

Comment:
We don't differentiate the others on platform, I think it's fair to call this one just `PTMessagEase` (and treat iOS as the special case worth calling out).